### DISCUSSION TITLE
Movement effects are not working

### DISCUSSION THREAD

**Discussion by spitfire1081** [DFreds/dfreds-convenient-effects#319] [Ideas]
When i set RESTRAINED to a token, the player is still able to move it.
I checked the DAE information and it appears to be ok, setting the movement to 0. The ruler integration shows red too, but the player still can move the token.

**Comment by spitfire1081** [comment]
The idea is to prevent the movement lol

**Comment by thatlonelybugbear** [comment]
That would need another module or at least a world script to offer that functionality.

**Comment by DFreds** [comment]
See #320
